### PR TITLE
V2 docs improvements

### PR DESCRIPTION
**Pull request by SimonClark**
Changes to make the docs look and work a bit better...  Let me know what you think / like.

image

### CODE REVIEW

**Comment by z3dev** [comment]
@SimonClark this is good, but I can see where this is heading.

My first reaction is that anyone should be able to add to documentation, especially non-API documentation. I’m a HUGE fan of WIKI, and think that users can contribute changes or content very easily. I’m not so sure about MD files.

Having said that, I’m open to experimenting.

**Comment by SimonClark** [comment]
> My first reaction is that anyone should be able to add to documentation, especially non-API documentation. I’m a HUGE fan of WIKI, and think that users can contribute changes or content very easily. I’m not so sure about MD files.

Yeah, I think wiki based documentation is good goal.  Eventually we need something more robust than jsdoc, or that works with jsdoc to add comments and wiki functionality. I've got some ideas.

This PR is a vanity piece for me. The default style and navigation bugs the crap out of me.  Adding the tutorials and homepage content seemed like a good thing too.

**Comment by SimonClark** [comment]
Now, that said, there is a "Disqus" option in the docdash theme.  It would be somewhat trivial to add on-page discussions.  But... if/when we moved to something more customized, we might lose all existing comments.

**Comment by SimonClark** [comment]
I think more valuable than user-editable docs, in the short term, would be a user-contributable script repository.  Something like Thingiverse, but for jscad scripts.

File: jsdoc.json [hunk 1]
```diff
@@ -34,6 +34,67 @@
...
+    "removeQuotes": ["none"],
+    "scripts": ["jscad-docs.css", "jscad-docs.js"],
+    "menu": {
+      "openjscad.org (v1)": {
+        "href":"https://openjscad.org",
+        "target":"_blank",
+        "class":"menu-item",
+        "id":"website_link"
+      },
+      "JSCad User Group": {
```
Comment (z3dev) [inline jsdoc.json:86]: Please add V2 as well. Website is www.jscad.xyz

File: jsdoc/assets/README.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
+# JSCAD V2 - Documentation
```
Comment (z3dev) [inline jsdoc/assets/README.md:1]: Can this be cleaned up a little? For example, screen shots don't really apply here.

File: jsdoc/assets/static/jscad-docs.css [hunk 3]
```diff
@@ -0,0 +1,187 @@
+body {
```
Comment (z3dev) [inline jsdoc/assets/static/jscad-docs.css:1]: HA HA HA... you really don't like the default colors.

File: packages/examples/old/various/logo.js [hunk 4]
```diff
@@ -5,20 +5,27 @@
 // tags       : Logo,Intersection,Sphere,Cube,Big
```
Comment (z3dev) [inline packages/examples/old/various/logo.js:5]: Why this change?

**Review by z3dev — CHANGES_REQUESTED** [review]
I have some suggestions for a few small changes. Otherwise, ready to go.

File: jsdoc.json (hunk 1 above)
Comment (SimonClark) [reply jsdoc.json:86]: Cool!  Will do.

File: jsdoc/assets/README.md (hunk 2 above)
Comment (SimonClark) [reply jsdoc/assets/README.md:1]: Yeah.  I'd not turned an eye towards this file yet. Coming soon...

File: jsdoc/assets/static/jscad-docs.css (hunk 3 above)
Comment (SimonClark) [reply jsdoc/assets/static/jscad-docs.css:1]: it's, uh, well. No. No I don't. Not since 1995.

File: packages/examples/old/various/logo.js (hunk 4 above)
Comment (SimonClark) [reply packages/examples/old/various/logo.js:5]: Hah, used it to generate the background.  I'll revert

**Review by z3dev — APPROVED** [review]
Good to go...
